src/Payments.sol

```diff
@@ -1754,6 +1783,43 @@ contract Payments is ReentrancyGuard {
 
         return (fundedUntilEpoch, currentFunds, availableFunds, currentLockupRate);
     }
+
+    /**
+     * @notice Burn FIL to buy the network fees
+     * @param token Which kind of fees to buy
+     * @param recipient Receives the purchased fees
+     * @param requested Exact amount of fees transferred
+     */
+    function burnFILForFees(IERC20 token, address recipient, uint256 requested) external payable {
+        Account storage fees = accounts[token][address(this)];
+        uint256 available = fees.funds;
+        require(available >= requested, Errors.WithdrawAmountExceedsAccumulatedFees(token, available, requested));
+
+        AuctionInfo storage auction = auctionInfo[token];
+        uint256 auctionPrice = uint256(auction.startPrice).decay(block.timestamp - auction.startTime);
+        require(msg.value >= auctionPrice, Errors.InsufficientNativeTokenForBurn(msg.value, auctionPrice));
+
+        auction.startPrice = uint88(auctionPrice * Dutch.RESET_FACTOR);
+        auction.startTime = uint168(block.timestamp);
+
+        fees.funds = available - requested;
+
+        (bool success,) = BURN_ADDRESS.call{value: msg.value}("");
```

Comment:
I'd transfer with method 0, for gas and safety (assuming we're sending to f099).

Reply:
what does this mean?

Reply:
`0xff00000000000000000000000000000000000063` is f099.

Reply:
@ZenGround0 yes. TBH, it's probably fine either way.

Reply:
That's not an issue as precompiles work fine in tests if you set things up: https://github.com/FilOzone/pdp/blob/main/src/PDPVerifier.sol#L615 

I think its probably not a real safety issue (though please correct me if I'm wrong)

But @wjmelements since you are very motivated to reduce gas costs it is worth looking into the difference.

Reply:
I couldn't help exploring this a little further to understand the suggestion: https://github.com/FilOzone/filecoin-pay/pull/234

As per comment in there, I'm not sure it's a better option here. Maybe it ends up being better in gas (I kind of doubt it), but how much is gas going to factor into settlement price for the auction so do we care?

Reply:
I tried measuring gas with my `forge script` method but I hit this foundry bug. https://github.com/foundry-rs/foundry/issues/11731

So let's make this a separate PR. I will create an issue to track it.